### PR TITLE
translate language codes

### LINKED ISSUE #94: translate country codes for "main languages" into words
translate country codes for "main languages" into words, e.g. "EN" into "English" on the event details page

### PR DESCRIPTION
**Pull request by smierz**
Hello you two, 
sorry, my browser just crashed, while I  was saying that i think the code is finished :=)
( fixes #94 )

Not sure about the formatting though, because it is 
German/French DE/FR and not German (DE), French (FR)
Let me know if this needs fixing

### CODE REVIEW

**Comment by weeman1337** [comment]
Thank you. I will have a look.

File: generator/consts.py [hunk 1]
```diff
@@ -34,13 +34,15 @@
 iso_639_languages = {
     "BG": "Bulgarian",
     "BS": "Bosnian",
-    "CS": "Czech",
+    "CA": "Catalan",
+    "CZ": "Czech",
```
Comment (weeman1337) [inline generator/consts.py:38]: We are using the ISO-639-1 codes here.
[Czech](https://de.wikipedia.org/wiki/Tschechische_Sprache) has the code `CS`.

If our data is wrong in the csv files we should correct that.

File: generator/consts.py [hunk 2]
```diff
@@ -34,13 +34,15 @@
...
     "DA": "Danish",
     "DE": "German",
     "EN": "English",
     "ES": "Spanish",
     "FI": "Finnish",
     "FR": "French",
+    "GR": "Greek",
```
Comment (weeman1337) [inline generator/consts.py:45]: This should be `EL`. As mentioned in another comment: if the csv is wrong we should correct it there.

**Comment by weeman1337** [comment]
> German/French DE/FR and not German (DE), French (FR)

While I would prefer German [DE], French [FR] I couldn't find any example of that online. We can keep it like you suggested in the PR since this seems to be a corner case.

**Comment by weeman1337** [comment]
I think it's fine so far. Can you please check the ISO codes as mentioned in the comments @smierz ? After that I will merge this PR :+1:

**Comment by smierz** [comment]
I changed the values in the data files to conform to iso 639
but also found it easier to use a library 

- for checking values and in case the values in the data do not conform print a warning
- translating code to names instead of maintaining a list in consts

the only "ugly" thing is for greek (EL) the iso-conform name translates to "Modern Greek (1453-)"

**Comment by smierz** [comment]
Additionally I set the default for main_language to an empty string instead of "?" which should solve #239

**Comment by weeman1337** [comment]
Happy new year :smile: and thanks for the update. Looks fine.

**Pull request merged**